geth/jail/handlers.go

```diff
@@ -51,6 +52,11 @@ func registerHandlers(jail *Jail, cell common.JailCell, chatID string) error {
 		return err
 	}
 
+	// register accounts handler
+	if err = registerHandler("accounts", makeAccountsHandler(jail, cell)); err != nil {
```

Comment:
I don't get it. In `StatusBackend` we have:
```go
rpcClient.RegisterHandler("eth_accounts", m.accountManager.AccountsRPCHandler())
rpcClient.RegisterHandler("eth_sendTransaction", m.txQueueManager.SendTransactionRPCHandler)
```

When `web3.eth.accounts` is called, it should do an RPC call and go through the already registered handler.